jsonnet/telemeter/metrics.jsonnet

```diff
@@ -94,4 +94,14 @@
   '{__name__="job:ceph_pools_iops_bytes:total"}',
   // job:ceph_versions_running:count is the total count of ceph cluster versions running.
   '{__name__="job:ceph_versions_running:count"}',
-]
+  // job:NooBaa_buckets_health:sum is the number of unhealthy buckets
+  '{__name__="job:NooBaa_buckets_health:sum"}',
+  // job:NooBaa_bucket_count:sum is the number buckets
```

Comment:
Can this be rephrased ? Dosen't make complete sense

Reply:
Can you suggest a more suitable name?